### ISSUE TITLE
MacOSX 10.12 Fix test_list_record_types test

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt-jenkins#389]
Currently the test `integration.modules.test_libcloud_dns.LibcloudDNSTest.test_list_record_types` is failing on MacOSX 10.12 on the head of nitrogen with the following error:

```
Error Message

False is not true
Stacktrace

Traceback (most recent call last):
  File "/testing/tests/integration/modules/test_libcloud_dns.py", line 16, in test_list_record_types
    self.assertTrue('SPF' in self.run_function('libcloud_dns.list_record_types', ['profile_test1']))
AssertionError: False is not true
		
Standard Output

	
Standard Error
```

**Comment by gtmanfred** [comment]
I am also seeing this pass, will see if it fails here http://mac-jenkins.lehi.saltstack.net:8080/job/2017.5/job/macOS-10.12/124/console

**Comment by Ch3LL** [comment]
I was able to replicate the failure. I will build you a box so you can replicate it there and send you the details later.

**Comment by gtmanfred** [comment]
Fixed in #41657
